ReText/window.py

```diff
@@ -211,8 +211,8 @@ def __init__(self, parent=None):
 			trig=lambda: self.insertChars('*'))
 		self.actionUnderline = self.act(self.tr('Underline'), shct=QKeySequence.Underline,
 			trig=lambda: self.insertTag('u'))
-		self.usefulTags = ('a', 'big', 'center', 'img', 's', 'small', 'span',
-			'table', 'td', 'tr', 'u')
+		self.usefulTags = ('header', 'italic', 'bold', 'numbering', 'bullets', 'image', 'link',
```

Comment:
yes, combobox + key shortcut.
Maybe the problem is make them clear for users, I think that the shortcut should be declared in a help page.